### ISSUE TITLE
Pasting a negative value over a negative value that as a currency symbol and its numbers selected throws an error

### ISSUE THREAD

**Issue by sbusch** [autoNumeric/autoNumeric#593]
### Current behavior

Can't paste a negative value into a autoNumeric `<input>` (french() Preset) already containing a negative value. Value is unchanged, JS error thrown.

### Expected behavior

Pasted value replaces current value.

### Steps to reproduce the problem

1. Use autoNumeric version `4.2.15` (current)
2. In the browser `Chrome` version `67.0.3396.99` (current)
3. On the `macOS` operating system
4. Then instantiate the autoNumeric object with the following options:
```js
new AutoNumeric(domElement).french();
```
5. Focus  the element with `<Tab>` key. Only the value portion, without the currency, should now be auto-selected
6. Paste a negative value, e.g. -5
7. Value in input box is unchanged, error thrown in JS console:

```
Uncaught Error: The pasted value '3' would result into an invalid content '3-'.
    at Function.value (autonumeric@4.2.15:1)
    at e.value (autonumeric@4.2.15:1)
    at HTMLInputElement._onPasteFunc (autonumeric@4.2.15:1)
```

### Link to live example

[Live-Example on CodeSandbox](https://codesandbox.io/s/x2yox22jlo)

Update: used wrong link for live-example, sorry

**Comment by AlexandreBonneau** [comment]
Thanks for the detailed bug report and reproducible test case.
I'm a bit swamped currently, but should be able to check on AutoNumeric bug reports in a few weeks!

**Comment by AlexandreBonneau** [comment]
This is fixed in `v4.3.3`
